include/net/lwm2m.h

```diff
@@ -207,6 +207,9 @@ int lwm2m_engine_register_post_write_callback(char *path,
 int lwm2m_engine_register_exec_callback(char *path,
 					lwm2m_engine_exec_cb_t cb);
 
+/* lwm2m bit conversion macro */
+#define LWM2M_BIT(val)			(1 << val)
```

Comment:
I just wonder do we really need this macro as zephyr already has BIT() macro

Reply:
Great catch.  We needed one before because of LWM2M_OP_NONE which was removed in this patchset.  We can use the generic macro now.  I'll fix.